### DISCUSSION TITLE
post/redesign-blog

### DISCUSSION THREAD

**Discussion by giscus** [bepyan/bepyan.me.v2#3] [General]
# post/redesign-blog

심플함으로 돌아가다.

https://bepyan.me/post/redesign-blog

**Comment by bocastle** [comment]
구글링하다 여기까지 오게되었습니다.
좋은글 보고 댓글 등록하고 갑니다.

**Reply by bepyan** [reply]
먼 길에서 찾아와주셔서 감사합니다!
글이 도움이 되길 바래요 :)

**Comment by kyh196201** [comment]
블로그 너무 깔끔하고 좋네요!
소스 공유도 너무 감사합니다 :-) 많이 참고할게요!

**Reply by bepyan** [reply]
좋게 봐주셔서 감사합니다 ㅎㅎ
빈틈 투성인 코드이지만,, 도움이 되길 바래요~!

**Comment by kimsuyeon0916** [comment]
블로그를 제작하던 와중에 이전 블로그를 통해서 들어오게 되었습니다! 좋은 글 올려주셔서 참고가 많이 되고 있습니다 감사합니다!😊

**Reply by bepyan** [reply]
제 글이 도움이 되어 기쁘네요 ☺️
앞으로 꾸준히 좋은 글을 써보도록 노력해볼게요!

**Comment by chebread** [comment]
Bepyan님 와.. 블로그가 정말 이쁘네요.. 프론트엔드 개발자로서 진짜 너무 만들고 싶은 블로그입니다!

**Reply by bepyan** [reply]
감사합니다! @chebread님의 멋진 여정 또한 응원합니다-!

**Comment by nyongho11** [comment]
설계에서 진지하게 고찰하고 섬세하게 다가가는 모습.. 👍 저도 개인 블로그 제작 중인데 평안님 게시물보고 많은 도움 얻고 있습니다. 항상 감사합니다~

**Reply by bepyan** [reply]
댓글 감사합니다 :) 용호님의 멋진 블로그 또한 응원합니다!

**Comment by ZETA-A** [comment]
블로그를 만들고 갈아엎고 다시 만들다가 갈아엎고... 수많은 과정을 거치다 평안님의 블로그를 보고 감동 받았습니다. 감사합니다!

**Comment by leeworms** [comment]
블로그 디자인에 감명받고, 글 솜씨에 더 감명받았습니다..!
소스 공유도 너무 감사드립니다! 
덕분에 아주 예쁜 블로그 하나 장만 했습니다ㅎㅎ

**Comment by vaynel** [comment]
매우 잘읽었습니다
